### PR TITLE
[The Trade Desk] Hide batching fields

### PR DESCRIPTION
**Pull request by maryamsharif**
This PR is to make some fields in TTD read_only and hidden as they should not be edited by customers.

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

BEFORE
Screenshot 2023-08-28 at 4 41 26 PM

AFTER
Screenshot 2023-08-28 at 4 51 15 PM

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [ ] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [x] [Segmenters] Tested in the staging environment

### CODE REVIEW

**Review by marinhero — APPROVED** [review]
Great teamwork here at play @maryamsharif and @nick-Ag. Fantastic job.

File: packages/destination-actions/src/destinations/the-trade-desk-crm/properties.ts [hunk 1]
```diff
@@ -30,50 +30,36 @@ export const pii_type: InputField = {
...
-  }
+  },
+  readOnly: true
 }
 
 export const event_name: InputField = {
   label: 'Event Name',
   description: 'The name of the current Segment event.',
-  type: 'hidden', // This field is hidden from customers because the desired value always appears at path '$.event' in Personas events.
+  type: 'hidden',
```
Comment (nick-Ag) [inline packages/destination-actions/src/destinations/the-trade-desk-crm/properties.ts:43]:
```suggestion
  type: 'string',
  unsafe_hidden: true,
```
I'll be migrating usages of `type: hidden` at some point so we can just make this change now

**Review by nick-Ag — APPROVED** [review]
Looks good to me, thanks!

**Comment by joe-ayoub-segment** [comment]
hi @maryamsharif this PR has been deployed

**Pull request merged**